devnets/chainSpec.ts

```diff
@@ -66,4 +76,7 @@ interface GenesisConfig {
   balances: {
     balances: [account: string, initialBalance: number][]
   }
+  session?: {
+    keys: [account: string, account: string, key: SessionKey][]
+  }
```

Comment:
@harrysolovay not every runtime uses `pallet_session` and some runtimes use a different key for this pallet.
Some runtimes configure it as `palletSession` or `authorMapping`.
Furthermore, the pallet is generic over the `SessionKey`.
For example
```rust
// rococo
impl_opaque_keys! {
	pub struct SessionKeys {
		pub grandpa: Grandpa,
		pub babe: Babe,
		pub im_online: ImOnline,
		pub para_validator: Initializer,
		pub para_assignment: ParaSessionInfo,
		pub authority_discovery: AuthorityDiscovery,
		pub beefy: Beefy,
	}
}

// polkadot
impl_opaque_keys! {
	pub struct SessionKeys {
		pub grandpa: Grandpa,
		pub babe: Babe,
		pub im_online: ImOnline,
		pub para_validator: Initializer,
		pub para_assignment: ParaSessionInfo,
		pub authority_discovery: AuthorityDiscovery,
	}
}
```

Shall we leave it as is?

Reply:
Unless this is currently blocking use of a particular test network / breaking our examples, let's worry about making this configurable later on. Can you please open an issue for this so that we can track it?

Reply:
created https://github.com/paritytech/capi/issues/889